packages/codemods/src/codemods/v0.39.x/updateCellMocks/updateCellMocks.yargs.ts

```diff
@@ -0,0 +1,36 @@
+import path from 'path'
+
+import fg from 'fast-glob'
+import task from 'tasuku'
+
+import getRWPaths from '../../../lib/getRWPaths'
+import runTransform from '../../../lib/runTransform'
+
+export const command = 'update-cell-mocks'
+export const description =
+  '(v0.38->v0.39) Updates standard cell mocks to export functions, instead of objects'
+
+export const handler = () => {
+  const rwPaths = getRWPaths()
+
+  const cellMocks = fg.sync('**/*.mock.{js,ts}', {
+    cwd: rwPaths.web.src,
+    absolute: true,
+  })
+
+  task(
+    'Updating Cell mocks',
+    async ({ setWarning, setOutput }: task.TaskInnerApi) => {
+      if (cellMocks.length <= 1) {
```

Comment:
I think this needs to be:
```suggestion
      if (cellMocks.length < 1) {
```